### PR TITLE
Fix privacy link

### PR DESCRIPTION
**Pull request by HaoK**
Fix for https://github.com/aspnet/AspNetCore/issues/9403

Conditionally render Privacy link based on whether the file exists (also fix the area link to work)

### CODE REVIEW

File: src/Identity/UI/src/Areas/Identity/Pages/V4/_Layout.cshtml [hunk 1]
```diff
@@ -56,7 +56,14 @@
     </div>
     <footer class="footer border-top pl-3 text-muted">
         <div class="container">
-        &copy; @DateTime.Now.Year - @Environment.ApplicationName - <a asp-area="" asp-page="Privacy">Privacy</a>
+        &copy; @DateTime.Now.Year - @Environment.ApplicationName -
+            @{
```
Comment (Eilon) [inline src/Identity/UI/src/Areas/Identity/Pages/V4/_Layout.cshtml]: I find it odd to have a runtime check for this file in here. That's certainly not what any real app would do. Maybe it's best to omit the link entirely, and then it's up to the app to override the layout file to put in the privacy link if they want it?

Comment (HaoK) [reply src/Identity/UI/src/Areas/Identity/Pages/V4/_Layout.cshtml]: @blowdart are you ok if we just omit the privacy link from the identity UI login page?  Its already only there in V4 apps.  So removing it just gets us back to the V3 situation anyways.

The privacy link is still in the main index page, it just would be omitted from identity UI pages

Comment (blowdart) [reply src/Identity/UI/src/Areas/Identity/Pages/V4/_Layout.cshtml]: No. This has GDPR implications. The privacy link can't just vanish from pages, especially pages that take PII. It needs to be everywhere.

Comment (Eilon) [reply src/Identity/UI/src/Areas/Identity/Pages/V4/_Layout.cshtml]: Was just expressing my opinion. Ultimately the Identity UI is kind of a weird case of pre-packaged UI, so there aren't really established rules for it. Blaze your own path :)

**Comment by HaoK** [comment]
@ajcvickers for FYI in case this is something for preview9

File: src/Identity/UI/src/Areas/Identity/Pages/V4/_Layout.cshtml (hunk 1 above)
Comment (javiercn) [reply src/Identity/UI/src/Areas/Identity/Pages/V4/_Layout.cshtml]: I think it’s fine. This doesn’t show up anywhere else but in our implementation.

We do a similar thing to detect login partial and throw a specific error if missing.

**Review by javiercn — APPROVED** [review]
Does the V3 version not have this?

**Comment by javiercn** [comment]
I believe it's preview9 or never

**Comment by HaoK** [comment]
Yeah V3 doesn't have it today, shrug https://github.com/aspnet/AspNetCore/blob/master/src/Identity/UI/src/Areas/Identity/Pages/V3/_Layout.cshtml#L56

**Comment by pranavkm** [comment]
@HaoK does this need to go in preview9?

**Comment by HaoK** [comment]
Yeah I guess this needs to be considered for preview 9, cc @ajcvickers

**Comment by ajcvickers** [comment]
@HaoK You'l need to create a PR against the 3.0 branch (or the preview 9 branch if that exists before you start) put it in the 3.0 milestone, and fill out the ask mode template: https://microsoft.sharepoint.com/teams/netfx/CLR/ProjectK/_layouts/OneNote.aspx?id=%2Fteams%2Fnetfx%2FCLR%2FProjectK%2FDocuments%2FTactics%2FNET%20Core%20Release%2Fnet-core-release&wd=target%28net-core-release.one%7C0CF34A20-F342-FA43-8944-18B555589A6B%2FShiproom%20Info%20and%20Templates%7C30A2770F-73EF-A74D-9C01-0D0B4BEAC431%2F%29

Then email will need to be sent to tactics to get approval.

**Comment by HaoK** [comment]
PR created here https://github.com/aspnet/AspNetCore/pull/13262

I filled out the ask mode template I think.  @blowdart can you start a mail to tactics for this?

**Comment by HaoK** [comment]
release PR was merged, closing since release changes will flow back to master eventually

**Pull request closed without merging**